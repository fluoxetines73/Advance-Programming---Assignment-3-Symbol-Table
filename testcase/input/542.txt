PRINT
BEGIN
INSERT e_123_2r number
LOOKUP e_123_2r
INSERT x number
INSERT g_1 number
END