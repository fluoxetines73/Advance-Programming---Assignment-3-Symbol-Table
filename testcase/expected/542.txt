
success
1
success
success